x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -103,11 +100,15 @@ export class HeadlessChromiumDriverFactory {
         proxyConfig: this.browserConfig.proxy,
       });
 
-      let browser: puppeteer.Browser;
-      let page: puppeteer.Page;
+      let browser: Browser;
+      let page: Page;
+
+      // The inspector works over websockets,
+      // so we need to conditionally interact
+      // with chrome via pipes depending on this flag
       try {
-        browser = await puppeteer.launch({
-          pipe: true,
+        browser = await launch({
+          pipe: !this.browserConfig.inspect,
           userDataDir,
           executablePath: this.binaryPath,
           ignoreHTTPSErrors: true,
```

Comment:
Is there a way to type-check `chromiumArgs`? `verboseLogging` is not a valid field: https://github.com/elastic/kibana/issues/41635

Reply:
Well, I guess not, since `args` is not very strictly typed: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/puppeteer/index.d.ts#L2059

Reply:
I'll try and address that as well